app.js

```diff
@@ -3,14 +3,36 @@ const app = express();
 
 const {
     getEndpoints,
-    getTopics
+    getTopics,
+    getArticlesById,
   } = require('./controllers/news-controller');
 
 app.use(express.json());
 
 app.get('/api', getEndpoints)
 app.get('/api/topics', getTopics)
+app.get('/api/articles/:article_id', getArticlesById)
 
-module.exports = app;
+app.use((err, req, res, next) => {
+    // postgres errors
 
-//
\ No newline at end of file
+    if(err.code == "22P02") {
+      res.status(400).send({ msg: "Bad request" })
+    }
+    else if(err.code == "23502") {
+      res.status(400).send({ msg: 'Bad request'})
+    }
```

Comment:
You have two codes here that respond with the same msg - use || to combine the if statements (and also use strict ===)